spec/test-data/translations/glotpress-download/expected/values-fr/strings.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-31 09:54:04+0000
+Plural-Forms: nplurals=2; plural=n > 1;
+Generator: GlotPress/2.4.0-alpha
+Language: fr
+-->
+<resources>
+    <string name="product_attributes_error_saving">Erreur lors de l’enregistrement de vos attributs</string>
+    <string name="shipping_label_payments_cant_edit_warning">Seul le propriétaire du site peut gérer les moyens de paiement des étiquettes d’expédition. Veuillez contacter le propriétaire de la boutique %1$s (%2$s) pour gérer les moyens de paiement.</string>
+    <string name="shipping_label_shipping_method_free_shipping">Livraison gratuite</string>
+    <string name="shipping_label_shipping_method_local_pickup">Point de vente</string>
+    <string name="shipping_label_shipping_method_flat_rate">Forfait</string>
+    <string name="shipping_label_shipping_carrier_flat_fee_banner_message">Le client a payé %1$s (%2$s) pour la livraison</string>
```

Comment:
Figured it was nice to include at least one key with `%n$s` placeholders to ensure they are not messed up for some reason either.